.github/CODEOWNERS

```diff
@@ -104,23 +104,17 @@ ddtrace/internal/remoteconfig       @DataDog/remote-config @DataDog/apm-core-pyt
 tests/internal/remoteconfig         @DataDog/remote-config @DataDog/apm-core-python
 
 # Guild
-# Note that core is added here since 2 reviewers are required. The guild isn't
-# big enough to support if a guild member makes a change which means 2 other
-# guild members have to review.
-# We should enforce this for important files, but more frequently edited ones
-# can settle for having 1 guild member review + a core team member.
 setup.py                            @DataDog/python-guild
 setup.cfg                           @DataDog/python-guild
 pyproject.toml                      @DataDog/python-guild
 hatch.toml                          @DataDog/python-guild
-.readthedocs.yml                    @DataDog/python-guild  @DataDog/apm-core-python
-README.md                           @DataDog/python-guild  @DataDog/apm-core-python
-mypy.ini                            @DataDog/python-guild  @DataDog/apm-core-python
-.github/PULL_REQUEST_TEMPLATE.md    @DataDog/python-guild  @DataDog/apm-core-python
-.github/ISSUE_TEMPLATE.md           @DataDog/python-guild  @DataDog/apm-core-python
-.github/CODEOWNERS                  @DataDog/python-guild  @DataDog/apm-core-python
-.github/workflows/system-tests.yml  @DataDog/python-guild  @DataDog/apm-core-python
```

Comment:
Removed core because the guild is a super set of the core team members

Reply:
That's not the case, though. The guild only has two core members.

Reply:
Oh true 🤔 OK I'll add it back for now until we can make this a true statement 😄